tests/alerts/test_honeycomb_case.py

```diff
@@ -0,0 +1,74 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+
+from positive_alert_test_case import PositiveAlertTestCase
+from negative_alert_test_case import NegativeAlertTestCase
+
+from alert_test_suite import AlertTestSuite
+
+
+class TestAlertHoneycomb(AlertTestSuite):
+    alert_filename = "honeycomb"
+
+    # This event is the default positive event that will cause the
+    # alert to trigger
+    default_event = {
+        "_type": "event",
+        "_source": {
+            "category": "syslog",
+            "details": {
+                "processname": "Honeycomb"
```

Comment:
My guess is that this default event needs summary data if the alert and aggregation is consuming this information.

We should probably also consider providing two alternative examples with one from say IP A and another from IP B to ensure that our aggregated source IPs are properly being served.

Additionally, as we noted in testing, there's a weird quote chars being presented in the summary from honeycomb and we should carry these forward in our test cases.